### DISCUSSION TITLE
FSI simulation only without optimization

### DISCUSSION THREAD

**Discussion by shaheidula** [mdolab/dafoam#427] [General]
Dear Prof. Ping, 
Can we run only FSI simulation without any optimization?

**Comment by friedenhe** [comment]
For steady-state FSI, we can do optimization. For unsteady FSI, we can do simulations only.

**Comment by shaheidula** [comment]
Dear Prof. Ping, 
Thanks a lot for your reply. To run the FSI simulation only, do we just need to replace the task “opt” by task "runPrimal" or  "runAdjoint"? 
I could run the steady-state FSI based optimization but I would like to run the FSI simulation only. Do we have any tutorial or hint on how to construct the runScript for this one? 
Thanks a lot,

**Reply by friedenhe** [reply]
>To run the FSI simulation only, do we just need to replace the task “opt” by task "runPrimal" or "runAdjoint"?

Yes
